models/index.js

```diff
@@ -0,0 +1,27 @@
+const Sequelize = require('sequelize')
+const authorsModel = require('./authors')
+const genresModel = require('./genres')
+const novelsModel = require('./novels')
+const novelsGenresModel = require('./novelsGenres')
```

Comment:
You named the file singular `novelGenre` which causes this import to fail and thus your app throws errors which is a huge chunk of points to lose sadly.